src/functions/template-tags/month.php

```diff
@@ -195,6 +196,17 @@ function tribe_events_month_has_events() {
 		return apply_filters( 'tribe_events_month_has_events', false );
 	}
 
+	/**
+	 * Returns whether there are any events in the month,
+	 * with the filtered results.
+	 *
```

Comment:
Add a `@since`

Reply:
This function was already there. I will add the since of the version where it was introduced.